__tests__/app.tests.js

```diff
@@ -149,3 +149,17 @@ describe("GET api/article", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET articles with correct data type ", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.comments[0];
+        expect(output.author).toEqual(expect.any(String));
+        expect(output.created_at).toEqual(expect.any(String));
+        expect(output.article_id).toEqual(expect.any(Number));
+        expect(output.votes).toEqual(expect.any(Number));
+      });
+  });
```

Comment:
we need a 200 test for valid ID but no comments 
& can you think of some error handling tests?